autoscheduler/scheduler/create_schedules.py

```diff
@@ -102,8 +102,8 @@ def _get_meetings(course: CourseFilter, term: str, include_full: bool,
     # Get id for each valid section to filter and order meeting data
     # Also removes full sections if include_full is False
     sections = sections.values('id', 'current_enrollment', 'max_enrollment')
-    # If manually selected don't check if section is full before adding
-    if course.section_nums or include_full:
+    # if manually selected don't check if section is full before adding
```

Comment:
```suggestion
    # If manually selected, don't check if section is full before adding
```
Keep this capital like it was before since that's how we format python comments